### ISSUE TITLE
Rocket initial speed should depend on car speed

### ISSUE THREAD

**Issue by Obsidien** [agateau/pixelwheels#77]
I have been a bit surprised by a rocket bonus refusing to launch… until I’ve discovered that it *was* launched, but going at the exact same speed and in the exact same direction as my 2cv (that had taken an accelerator). To avoid this kind of things, I think the rocket should be launched to a speed that depends on the car speed, before taking its normal speed.

**Comment by agateau** [comment]
Good point, but it's a bit tricky because I don't want the rocket to be too fast, otherwise there is no way to dodge or escape it. Need to think about it :thinking:

**Comment by Obsidien** [comment]
It could take its normal speed soon after having been launched, I won’t see a problem with that I think. There’s just a need to ensure it has been visually separated from the car.

**Comment by agateau** [comment]
Good idea :+1:
